doc/material-controls-extensions.md

```diff
@@ -0,0 +1,38 @@
+# Material Control Extensions
+
+## Icon
+This feature allows to put an icon on some supported control. Those icons could be any kind of _Icon_: `<BitmapIcon />`, `<FontIcon />`, `<SymbolIcon/>`, or `<PathIcon />`.
+
+Here are supported control with samples:
+* TextBox:
+  ``` xml
+    <TextBox Style="{StaticResource MaterialFilledTextBoxStyle}">
+        <um:ControlExtensions.Icon>
+            <SymbolIcon Symbol="SolidStar" />
+        </um:ControlExtensions.Icon>
+    </ComboBox>
+  ```
+* ComboBox:
+  ``` xml
+    <ComboBox Style="{StaticResource MaterialComboBoxStyle}">
+        <um:ControlExtensions.Icon>
+            <SymbolIcon Symbol="SolidStar" />
+        </um:ControlExtensions.Icon>
+    </ComboBox>
+  ```
+
+## Alternate Content
+This feature allows to put a different content on a control when the state changes.

```

Comment:
What's the state property? Is it per control?

Reply:
control specific, and only currently ToggleButton uses it.